.github/workflows/housekeeping.yaml

```diff
@@ -18,28 +18,30 @@ jobs:
       pull-requests: write
 
     steps:
-      - uses: actions/stale@v9
+      - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639  # v9.1.0
```

Comment:
This action is whitelisted:
```
actions/* 
```

Therefore, we have the option to keep this action as it is.

Reply:
Done.